sdk/core/amqp-common/src/errors.ts

```diff
@@ -586,6 +605,10 @@ export function translate(err: AmqpError | Error): MessagingError {
       // not found
       error.retryable = false;
     }
+  } else if (isWebsocketError(err)) {
+    // Translate browser communication errors during opening handshake to generic SeviceCommunicationError
+    error = new MessagingError("Websocket connection failed.");
+    error.name = ConditionErrorNameMapper[ErrorNameConditionMapper.ServiceCommunicationError];
```

Comment:
When we do a `new MessagingError`, `retryable` on it is `true`. `ServiceCommunicationError` is not a retryable error.